client/src/app/components/home/home.component.html

```diff
@@ -1,2 +1,18 @@
-<app-dashboard></app-dashboard>
-<button (click)="logout()">Logout</button>
+<mat-drawer-container class="home-container">
+
+    <mat-drawer #drawer opened="true" mode="side">
+        <app-sidenav></app-sidenav>
+    </mat-drawer>
+
+    <mat-drawer-content class="home-content">
+        <button type="button" mat-button (click)="drawer.toggle()">
```

Comment:
The button is still not a mat-button by looking at its style in the page.
<img width="252" alt="Screen Shot 2020-10-11 at 7 15 33 PM" src="https://user-images.githubusercontent.com/35696935/95692758-2543e900-0bf6-11eb-8cf9-cb05fc7dcd4a.png">

Reply:
I think I will just change it to a regular button since it is only temporary anyway.